includes/Modules/Analytics_4/AMP_Tag.php

```diff
@@ -90,6 +115,12 @@ public function register() {
 	protected function render() {
 		$config = $this->get_tag_config();
 
+		if ( ! empty( $this->ads_conversion_id ) ) {
+			$config[ $this->ads_conversion_id ] = array(
+				'groups' => 'default',
+			);
+		}
+
 		$gtag_amp_opt = array(
 			'vars' => array(
 				'gtag_id' => $this->tag_id,
```

Comment:
Let's also reintroduce/preserve `optoutElementId` https://github.com/google/site-kit-wp/blob/67b4d23069487fc021d0dfadd80f87908895fd71/includes/Modules/Analytics/AMP_Tag.php#L118

Reply:
Updated, thank you!